src/components/fabSpeedDial/fabSpeedDial.spec.js

```diff
@@ -32,33 +32,37 @@ describe('<md-fab-speed-dial> directive', function() {
     expect(element.hasClass('md-right')).toBe(true);
   }));
 
-  it('opens when the trigger element is focused', inject(function() {
+  it('opens when the trigger element is focused', inject(function($timeout) {
     compileAndLink(
       '<md-fab-speed-dial><md-fab-trigger><button></button></md-fab-trigger></md-fab-speed-dial>'
     );
 
     element.find('button').triggerHandler('focus');
+    $timeout.flush();
```

Comment:
I don't think you need `$timeout.flush()` anymore.
A simple `$rootScope.$digest()` should be enough.